### ISSUE TITLE
Distribute points question reverts to 100

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#5924]
Received from user and confirmed as bug.
```
I am trying to set up a session and am running into a glitch. I have added a distribute points (among recipients) question. Every time I try change the total points to distribute to 25 x (number of recipients) when I click save changes, it reverts back to the 100 x (number of recipients). ?
```

**Comment by damithc** [comment]
@tarun14110 we need an immediate fix for this.

**Comment by damithc** [comment]
Before fixing the bug, write a failing test case to confirm the bug. This bug should never have made it to the wild if our test case design was good.

**Comment by chowyb** [comment]
Distribute points (among options)/(among recipients) questions appear to both be affected, and is caused by taking the numerical value of the "____ in total" input instead of the "____ X (number of options/recipients)".

This could be a temporary workaround while we fix the problem.

**Comment by damithc** [comment]
Yes, I've advised the user of this workaround but it is embarrassing :-p
If we did have a test case for this, it must have used the same value for both text boxes, thus hiding the bug. Variety of test values can improve the effectiveness of test cases.

**Comment by damithc** [comment]
@wkurniawan07 can release a hotfix for this ASAP, preferably sometime today? I haven't deployed the recent version yet. I'll wait for the hotfix version instead.

**Comment by chowyb** [comment]
Seems to be a bug introduced in #5602. While an additional input box for the numbers was added, both retained the same id and name, resulting in the first one being picked.

A (temporary) quick fix for now could be to add JS code to the frontend updating both input boxes to be the same (easy since the input boxes already call a function on change). A more proper fix would be to read in the number from the second input if "among recipients/among options" is selected, which may take slightly longer, but should not take more than a few hours.

I'll work on the proper fix on my own for now, but not assigning this to myself in case someone else wants to take up the issue.

**Comment by damithc** [comment]
Thanks for the quick response @chowyb :+1: 
@tarun14110 is a few hours behind us. So it's fine if you go ahead and fix it yourself.

**Comment by damithc** [comment]
BTW, PR against the release branch as this is a hotfix.

**Comment by chowyb** [comment]
Travis might not build from the PR though, almost done (save for a few other tests that may have been affected by the change), so I'll open to see if it'll build first.

**Comment by chowyb** [comment]
PR looks quite long as I branched off the most recent version of master instead of release, making it difficult to review. Are there any alternatives that might work better here?

**Comment by damithc** [comment]
@wkurniawan07 can we configure travis to build PRs against release branch?

**Comment by wkurniawan07** [comment]
I haven't seen it in action, but based on the `.travis.yml` configuration it should already be the case.

**Comment by chowyb** [comment]
@damithc @wkurniawan07 Travis is building the PR, but there remains the issue of 2 other commits not related to this issue in the PR. I might be able to shift the changes to a fresh branch off of release, but it will take about an hour.

**Comment by wkurniawan07** [comment]
Fixed in #5926.
